prime-angular-frontend/src/app/lib/utils/date-utils.class.ts

```diff
@@ -20,6 +20,23 @@ export class DateUtils {
     return moment(todayOrOtherDate).isAfter(minusDaysBeforeDate);
   }
 
+  /**
+ * @description
+ * Check that a date is within a number of days before another date.
+ */
+  public static withinDaysAfterDate(
+    date: string | Moment | null,
+    daysBeforeDate: number,
+    todayOrOtherDate: string | Moment = moment()
+  ): boolean {
+    if (!date || !todayOrOtherDate) {
+      return false;
+    }
+
+    const minusDaysBeforeDate = moment(todayOrOtherDate).subtract(daysBeforeDate, 'days');
+    return moment(date).isBefore(minusDaysBeforeDate);
```

Comment:
Is logic reversed?  If template last mod date is within 6 months of today, should this function return `true` or `false`?

Reply:
I guess the function name is confusing. Updated the function name, hope it makes more sense.

Reply:
Would it make more sense as `isDaysAfterDate` but switch the parameter order so that it would be called like `DateUtils.isDaysAfterDate(182, template.modifiedDate)`?

Reply:
Sure, the function is updated.